src/lib/falcor/FalcorRouter.js

```diff
@@ -1,53 +1,55 @@
 import BaseRouter from "falcor-router"
 import { ghostArticleQuery } from 'lib/ghostAPI'
-import { sqlArticleQuery, sqlAuthorQuery } from 'lib/sql'
+import { dbAuthorQuery, dbArticleQuery, dbAuthorArticleQuery, dbInfoPagesQuery, dbArticleIssueQuery,
+dbArticleAuthorQuery, dbLatestIssueQuery, dbCategoryQuery, dbCategoryArticleQuery,
+dbFeaturedArticleQuery, dbEditorPickQuery, dbIssueCategoryQuery, dbIssueQuery } from 'lib/mariaDB'
 import falcor from 'falcor'
+import _ from 'lodash';
 
-const $ref = falcor.Model.ref; 
+const $ref = falcor.Model.ref;
 
-// FLAG TO CHECK IF YOU WANT TO ACCESS GHOST API AND DATABASES
-// THIS IS JUST FOR DEVELOPMENT WHILE WE DON'T HAVE ACTUAL SERVERS SET UP
-const USE_DATABASES = false; 
-
-// Transform ghost API names into our names
-const mapGhostNames = (name) => {
-  switch(name) {
-    case 'pubDate':
-      return 'published_at';
-    case 'body':
-      return 'markdown';
+let mapGhostNames = (falcorName) => {
```

Comment:
I think we should refer to it as "teaser" whenever possible instead of "meta_description". Meta description isn't really clear for people who don't have much web development experience. 

I think the best solution is to abstract it away by just making any references "teaser" on the client side equate to "meta_description" for Ghost.